src/common.c

```diff
@@ -56,6 +62,22 @@ WasmStringSlice wasm_empty_string_slice(void) {
   return result;
 }
 
+WasmBool wasm_string_slice_eq(WasmStringSlice* s1, const char* s2) {
```

Comment:
Hm, might be nice to rename this (and/or `wasm_string_slices_are_equal`) so they are more consistent.